src/sentry/incidents/endpoints/organization_incident_index.py

```diff
@@ -47,6 +48,6 @@ def get(self, request, organization):
             queryset=incidents,
             order_by="-date_started",
             paginator_cls=OffsetPaginator,
-            on_results=lambda x: serialize(x, request.user),
+            on_results=lambda x: serialize(x, request.user, DetailedIncidentSerializer()),
```

Comment:
Is this so we can access `discoverQuery`? Might make more sense to move that down to `IncidentSerializer` instead of using the `DetailedSerializer` here, so that you don't need to fetch a bunch of extra data you don't need.

Reply:
@wedamija I'm looking to access the "alertRule" that DetailedSerializer has, should I instead add it to the IncidentSerializer?

Reply:
Yeah, I think we probably should. I think the only place we actually use the base serializer is the list endpoint anyway. When you move it, we should also start calling `serialize` on the batch of alert rules inside of `get_attrs`, rather than individually like we do now.